Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -4,13 +4,16 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Globalization;
```

Comment:
>Globalization [](http://example.com/codeflow?start=13&length=13)

Changes in this file drop in very comprehensive eventing. I have sent Suvam some data files offline that show what this looks like. Can provide to others as well.